### PR TITLE
ci: load scraper data

### PR DESCRIPTION
**Pull request by lukashartl**
## Basics

- [x] Details of what you changed are in commit messages.
- [x] References to issues, e.g. `close #X`, are in the commit messages.
- [ ] The buildserver is happy.
- [x] The PR is rebased with current master.

## Checklist

- [ ] I added unit tests for my code
- [ ] I fully described what my PR does in the documentation
      (not in the PR description)
- [ ] I fixed all affected documentation
- [ ] I fixed all affected decisions
- [ ] I added code comments, logging, and assertions as appropriate
- [ ] I translated all strings visible to the user
- [ ] I mentioned [every code or binary](/.reuse/dep5) not directly written or done by me in [reuse syntax](https://reuse.software/)

## Review

- [ ] I've tested the code
- [ ] I've read through the whole code
- [ ] Documentation is conforming to [our Documentation Guidelines](/doc/documentation.md)
- [ ] Examples are well chosen and understandable
- [ ] Code is conforming to our Coding Guidelines
- [ ] Code is consistent to [our Design Decisions](/doc/decisions)

### CODE REVIEW

**Comment by lukashartl** [comment]
@markus2330 loading data from the scraper works now :rocket:

File: ci/Jenkinsfile [hunk 1]
```diff
@@ -1,8 +1,13 @@
+def node_info() {
+    echo "Running this Stage on ${env.NODE_NAME}"
```
Comment (markus2330) [inline ci/Jenkinsfile:2]: Good to have this!

```suggestion
    echo "Running stage on ${env.NODE_NAME}"
```

File: ci/Jenkinsfile [hunk 2]
```diff
@@ -135,8 +150,18 @@ stage('Deploy PR') {
...
             sh "sudo /usr/local/bin/permaplant-deploy.sh pr ${WORKSPACE}"
+
+            echo 'Loading Data to PR'
+            dir('scraper') {
+                sh 'npm ci'
+                sh 'mkdir ./data/'
+                sh 'cp /nextcloud/mergedDatasets.csv ./data/'
+                sh 'npm run insert'
+                sh 'rm -rf ./data/'
+                sh 'rm -rf ./node_modules/'
```
Comment (markus2330) [inline ci/Jenkinsfile:163]: Seems to be unrelated to "Loading Data to PR"? Why was this needed?

**Review by markus2330 — APPROVED** [review]
Great job but it seems like the build did not run through.

**Comment by markus2330** [comment]
jenkins build libelektra please

**Comment by lukashartl** [comment]
jenkins build please

**Comment by lukashartl** [comment]
I didn't change anything, but it seems to work:

[Jenkins Job](https://build.libelektra.org/blue/organizations/jenkins/PermaplanT/detail/PR-331/4/pipeline/232/)

```
postgres@pr:~$ psql cidb

cidb=# SELECT COUNT(*) FROM plants;
 count 
-------
  9810
(1 row)
```

File: ci/Jenkinsfile (hunk 2 above)
Comment (lukashartl) [reply ci/Jenkinsfile:163]: I just added a manual cleanup, because in elektra Jenkins sometimes leaves old workspace files and those two folders are huge in relation to the very small containers 😸

**Comment by markus2330** [comment]
jenkins build please
